src/Domain/SalesforceWriteProxyRepository.php

```diff
@@ -65,8 +65,17 @@ public function pushAllPending(int $limit = 200): int
             ['id' => 'ASC'],
             $limit,
         );
+
         foreach ($proxiesToUpdate as $proxy) {
             $this->push($proxy, false);
+            if (getenv('APP_ENV') === 'regression') {
+                $createdAt = $proxy->getCreatedAt();
+                $this->logger->info(
+                    'Donation Id to update: ' . $proxy->getSalesforceId() .
```

Comment:
This won't tell us if there are newer ones beyond what the query reaches. Wouldn't it make more sense to query for the oldest and newest instead of following the existing query's limits?